ui/imports/shared/popups/SendModal.qml

```diff
@@ -86,13 +85,16 @@ StatusDialog {
         })
         property string resolvedENSAddress
         onIsReadyChanged: {
-            if(!isReady && stack.isLastGroup)
-                stack.back()
+            if(!isReady && isLastGroup)
+                stack.currentIndex = 0
         }
+        readonly property string uuid: Utils.uuid()
+        property bool isLastGroup: stack.currentIndex === 1
```

Comment:
Would there be a way to make this dynamic instead of hard coded? Something like `stack.currentIndex === stack.count - 1`

Reply:
There is already a `stack.isLastGroup` property

Reply:
@caybro what I am using is StackLayout not TransactionStackView.qml I will update it to the way @jrainville has recommended